index.js

```diff
@@ -1,131 +1,130 @@
-function detect() {
-  if (typeof navigator !== 'undefined') {
-    return parseUserAgent(navigator.userAgent);
-  }
-
-  return getNodeVersion();
-}
-
-function detectOS(userAgentString) {
-  var rules = getOperatingSystemRules();
-  var detected = rules.filter(function (os) {
-    return os.rule && os.rule.test(userAgentString);
-  })[0];
-
-  return detected ? detected.name : null;
-}
-
-function getNodeVersion() {
-  var isNode = typeof process !== 'undefined' && process.version;
-  return isNode && {
-    name: 'node',
-    version: process.version.slice(1),
-    os: process.platform
-  };
-}
-
-function parseUserAgent(userAgentString) {
-  var browsers = getBrowserRules();
-  if (!userAgentString) {
-    return null;
-  }
-
-  var detected = browsers.map(function(browser) {
-    var match = browser.rule.exec(userAgentString);
-    var version = match && match[1].split(/[._]/).slice(0,3);
-
-    if (version && version.length < 3) {
-      version = version.concat(version.length == 1 ? [0, 0] : [0]);
+(function (factory) {
```

Comment:
This file is compiled output - are you sure you want this in version control?

Reply:
Yeah, I was undecided about this (which is why I haven't responded to the comment). I will definitely remove the `index.d.ts` from the repo.  Given that this is a UMD compilation it's likely to be useful to someone who might not be using it in a commonjs environment (there have been a couple of people with this question).

I should investigate whether there has been someone clever who has made a package that does a nice postpublish action which automates creating a release in github and attaching things like this.  Then we can say something like "for a UMD build of `detect-browser` please see the release page".

Reply:
Well, as expected github definitely have an API endpoint for releases.  And [@semantic-release/github](https://github.com/semantic-release/github) looks interesting - will investigate.

Reply:
Tracking this in #83.